src/CMakeLists.txt

```
@@ -31,40 +43,79 @@ ExternalProject_Add(gflags-proj
 
 ExternalProject_Get_Property(gflags-proj install_dir)
 set(GFLAGS_ROOT ${install_dir})
-
 set(GFLAGS_INCLUDE_DIRS ${GFLAGS_ROOT}/include
         CACHE INTERNAL "Path to gflags include directory")
-
-set(GFLAGS_LIBRARY_PATH ${GFLAGS_ROOT}/lib/${CMAKE_FIND_LIBRARY_PREFIXES}gflags.a)
-set(GFLAGS_DEBUG_LIBRARY_PATH ${GFLAGS_ROOT}/lib/${CMAKE_FIND_LIBRARY_PREFIXES}gflags_debug.a)
+set(GFLAGS_LIBRARY_PATH ${GFLAGS_ROOT}/lib/libgflags${GFLAGS_WIN_LIB_SUFFIX}.a)
+set(GFLAGS_DEBUG_LIBRARY_PATH ${GFLAGS_ROOT}/lib/libgflags_debug.a)
```

Is `{GFLAGS_WIN_LIB_SUFFIX}` also necessary here?

---

It is - on Win, the static gflags lib is built as `libgflags_static.a`, whereas on Linux it's just `libgflags.a`

---

so it should be
```suggestion
set(GFLAGS_DEBUG_LIBRARY_PATH ${GFLAGS_ROOT}/lib/libgflags${GFLAGS_WIN_LIB_SUFFIX}_debug.a)
```
?

---

@the-joksim check comment above